gulpfile.js

```diff
@@ -324,7 +324,7 @@ function generateBootScript(manifest) {
 
   const defaultSidebarAppUrl = process.env.SIDEBAR_APP_URL
     ? `${process.env.SIDEBAR_APP_URL}`
-    : 'https://hypothes.is/app.html';
+    : 'http://localhost:5000/app.html';
```

Comment:
This will break the production build of the client since it uses the default sidebar app URL. See the "Publish to QA" section of Jenkinsfile for a way to set a non-default sidebar app URL for CI builds.

Reply:
It would be convenient if there was a way to enable SSL easily here. We can figure that out later though. I use SSL with locally signed certs for all of my testing as it is a more convenient way to test across a wide range of browsers and devices because configuring each device to accept a particular custom certificate is typically much easier than disabling mixed content warnings in every (browser, device).

Reply:
In the case of the client, that would be just a case of doing a custom `export SIDEBAR_APP_URL="https://localhost:5000/app.html"` in your shell, rignt?

Reply:
Yes, exactly.